src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -338,11 +338,13 @@ Assembly: 'SonarLint.VisualStudio.Roslyn.Suppressions, Version=6.0.0.0, Culture=
 Relative path: 'SonarLint.VisualStudio.Roslyn.Suppressions.dll'
 
 Referenced assemblies:
+- 'Microsoft.CodeAnalysis, Version=3.3.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

Comment:
Hmm, aren't we referencing 3.3.1?

Reply:
We're referencing v3.3.1 of the NuGet package. The assembly version number of both `Microsoft.CodeAnalysis.dll` and `Microsoft.CodeAnalysis.CSharp.dll` is v3.3.0.0:

![image](https://user-images.githubusercontent.com/10757559/156602022-7677202f-c837-4336-9679-4e2312451fbe.png)